syntax = "proto3";

package user_service;


service UserServices {
    rpc getUserInfo (UserId) returns (UserInfo) {}
}

message UserId {
    string id=1;
}

message UserInfo {
    string id = 1;
    string first_name = 2;
    string last_name = 3;
    string user_name = 4;
    string phone = 5;
    string email = 6;
    string role = 7;
}